### DISCUSSION TITLE
Need help in the implementation of gRPC server

### DISCUSSION THREAD

**Discussion by Jyoti-13** [loopbackio/loopback-next#8073] [Q&A]
I'm trying to implement gRPC server in which I am planning to create proto file for models and each methods in controller. Is there any better approach that I could use for implementing gRPC server?

**Comment by yeshamavani** [comment]
Hello @achrinza @raymondfeng 
This particular thread was started by one of my team mate. Help from loopback team for implementing gRPC server in a better way would be really appreciated.
